frontend/src/features/admin-form/common/mutations.ts

```diff
@@ -313,80 +313,8 @@ export const useMutateCollaborators = () => {
 export const useMutateFormPage = () => {
   const { formId } = useParams()
   if (!formId) throw new Error('No formId provided')
-
-  const queryClient = useQueryClient()
   const toast = useToast({ status: 'success', isClosable: true })
 
-  const updateStartPageFormData = useCallback(
-    (newData: StartPageUpdateDto) => {
-      queryClient.setQueryData(adminFormKeys.startPage(formId), newData)
-      // Only update adminForm if it already has prior data.
-      queryClient.setQueryData<AdminFormDto | undefined>(
```

Comment:
i think we will still need to update the query for the entire form, since this mutation will indeed update that portion

Reply:
oh yes, we should still do that